src/searchForm/SearchForm.tsx

```diff
@@ -59,7 +65,9 @@ const SearchForm = ({
   datepickerProps,
   stepperProps,
 }: SearchFormProps) => {
-  const mediaSize = useContext(MediaSizeContext)
+  // Use React.useContext syntaxt so we can mock it
+  // https://github.com/enzymejs/enzyme/issues/2176#issuecomment-533582429
+  const mediaSize = React.useContext(MediaSizeContext)
```

Comment:
Any reason why you removed `useContext` from the import to use it directly here?

Reply:
It's explained in the comment above. A line break before the comment will help to group them together maybe?

Reply:
Oh sorry I didn't see it, perfect then!